src/css/framework.css

```diff
@@ -0,0 +1,248 @@
+/* --- Variables --- */
+:root {
+
+	/* gutters play a major role in grids,
+	many declorations in the layout depend on the value set for the grid  */
+	--gutter-value: 2em;
```

Comment:
I think this should be rem not em.  The reasoning being that if a designer starts fiddling with font-sizes, then your layouts may change inadvertently.  

Ex:  If I change the font-size of a div to 2em and then nest a row with gutters, the width of the gutters would be off by 2em.